react/features/base/redux/MiddlewareRegistry.js

```diff
@@ -9,6 +9,7 @@ import type { Middleware } from 'redux';
  */
 class MiddlewareRegistry {
     _elements: Array<Middleware<*, *>>;
+    _elements2: Array<Middleware<*, *>>;
```

Comment:
I don't really like this. What is it trying to solve? Running stuff after all middlewares / reducers have run? Redux already has a pattern for it: `next(action)` and run your code after that. I'd rather stick with the well established patterns than to somewhat reinvent them.

Or did I miss something here?

Reply:
Yes, you are missing the fact that once `next(action)` is done it's to late for the "fatal error" feature to modify the action. That's because there are features like the external API which expect the state and the action to be in the final state after the `next(action)` call. I wanted to make the "fatal error" feature work as catch all, but in order to achieve that it needs to be executed after all regular middlewares in order to know if no other feature will claim the error handling.

In this commit I'm only trying to show what is needed. Probably it could be done in more generic way like for example allowing to set a priority to a middleware and then apply the middlewares after sorting (or doing insertion sort on register).